docs/source/topics/data_types.rst

```diff
@@ -26,7 +26,7 @@ The different data types can be accessed through the :py:func:`~aiida.plugins.fa
 
 A list of all the data entry points can be obtain running the command ``verdi plugin list aiida.data``.
 
-For all data types, you can follow the link to the corresponding data class in the API reference to read more about the class and its methods. We also detail what is stored in the database (mostly as attributes, so the information can be easily queried e.g. with the :ref:`QueryBuilder <how-to:data:find>`) and what is stored as a raw file in the AiiDA file repository (providing access to the file contents, but not efficiently queryable: this is useful for e.g. big data files that don't need to be queried for).
+For all data types, you can follow the link to the corresponding data class in the API reference to read more about the class and its methods. We also detail what is stored in the database (mostly as attributes, so the information can be easily queried e.g. with the :ref:`QueryBuilder <how-to:query>`) and what is stored as a raw file in the AiiDA file repository (providing access to the file contents, but not efficiently queryable: this is useful for e.g. big data files that don't need to be queried for).
```

Comment:
I am honestly a bit puzzled why all of these sections have multiple sentences per line. We may need to add a linter for this.

Reply:
> I am honestly a bit puzzled why all of these sections have multiple sentences per line. 

I don't know... perhaps copied from the old documentation?

> We may need to add a linter for this.

Feel free to add such a linter if that exists - might not be super straightforward to detect (dot/exclamation mark/... followed by upper case?) but probably you'll catch most cases.
I would say that should probably go into a separate PR.

Reply:
Oh yeah, this would need to go into a separate PR for sure. Just something I noticed while reviewing.